target/linux/ath79/dts/qca9558_tplink_archer-d7-v1.dts

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9558_tplink_archer-d7.dtsi"
+
+/ {
+	compatible = "tplink,archer-d7-v1", "qca,qca9558";
+	model = "TP-Link Archer D7 v1";
+};
+
+&mtdparts {
```

Comment:
I'm in favor of keeping the spi node together and would actually like to have it in both DTS files.